scania-digital-design-system/tegel
# feat(tabs): prefix change

**Describe pull-request**  
Updated prefix for folder, inline and navigation tabs.

**Solving issue**  
Fixes: https://tegel.atlassian.net/browse/DTS-1649

**How to test**  
1. Go to storybook.
2. Check in Components -> Folder, Inline & Navigation Tabs
3. Check the components APIs and styles.

**Suggested test steps**
